recipes/verilator/all/conandata.yml

```diff
@@ -9,6 +12,11 @@ sources:
     url: "https://www.veripool.org/ftp/verilator-4.034.tgz"
     sha256: "54ed7b06ee28b5d21f9d0ee98406d29a508e6124b0d10e54bb32081613ddb80b"
 patches:
+  "4.224":
+    - patch_file: "patches/0003-add-msvc_link-sh.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/0006-msvc-patches.patch"
```

Comment:
Have you considered opening a PR upstream with these changes?

It would be really nice to add the extra fields https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields so we can better track these 🙏

Reply:
I removed the mscv patches, since mscv builds are currently not supported anyway